stdlib/importlib/abc.pyi

```diff
@@ -81,7 +81,7 @@ if sys.version_info >= (3, 7):
         def contents(self) -> Iterator[str]: ...
 
 if sys.version_info >= (3, 9):
-    from typing import Protocol, runtime_checkable
+    from typing import runtime_checkable
```

Comment:
Another nit: the whole import could be merged with the top-level import.

Reply:
Are you sure? I don't think `typing.runtime_checkable` exists in older pythons.

Reply:
At least it's not gated in typeshed.

Reply:
But it was in fact new in 3.8. I suppose that's a bug, because it means type checkers won't catch imports of `typing.runtime_checkable` before 3.8. Changing `typing.pyi` to only export it after 3.8 might be annoying because we still have to mark a number of protocols in the same file as runtime-checkable.

In any case, you can definitely put it in the top-level import from `typing_extensions`.

Reply:
(Sorry I missed that you already opened an issue discussing this)